crates/dojo-lang/src/component.rs

```diff
@@ -167,6 +177,13 @@ pub fn handle_component_struct(
                 }
             }
 
+            #[cfg(test)]
+            impl $type_name$PrintImpl of PrintTrait<$type_name$> {
```

Comment:
```suggestion
            impl $type_name$PrintImpl of debug::PrintTrait<$type_name$> {
```

we can fully qualify it here to avoid the spurious imports